packages/broker/src/plugins/operator/VoteOnSuspectNodeHelper.ts

```diff
@@ -0,0 +1,54 @@
+import { Contract } from '@ethersproject/contracts'
+import { Provider } from '@ethersproject/providers'
+import { Signer } from '@ethersproject/abstract-signer'
+import { operatorABI } from '@streamr/network-contracts'
+import type { Operator } from '@streamr/network-contracts'
+import { Logger } from '@streamr/utils'
+import { OperatorServiceConfig } from './OperatorPlugin'
+
+export const VOTE_KICK = '0x0000000000000000000000000000000000000000000000000000000000000001'
+export const VOTE_NO_KICK = '0x0000000000000000000000000000000000000000000000000000000000000000'
+
+const logger = new Logger(module)
+
+export class VoteOnSuspectNodeHelper {
+    provider: Provider
+    address: string
+    contract: Operator
+    signer: Signer
```

Comment:
Access modifiers to enhance readability 
```suggestion
    private readonly provider: Provider
    private readonly address: string
    private readonly contract: Operator
    private readonly signer: Signer
```